### ISSUE TITLE
Darkmoon Faire -- Deathmatch -- Darkmoon treasure chest do not spawn at all... Achievement Darkmoon Duelist/Dominator

### ISSUE THREAD

**Issue by Nidhogg89** [gamefreedomgit/Maelstrom#6099]
It should spawn every 3 hours after midnight.

https://wowwiki-archive.fandom.com/wiki/Darkmoon_Treasure_Chest

https://wowwiki-archive.fandom.com/wiki/Darkmoon_Treasure_Chest)

**Comment by DusanJovanovic1991** [comment]
I'm an achievement farmer, can you please fix this one. It's the last achievement from Darkmoon Faire that doesn't work. Thank you in advance.

**Comment by Whirlwind89** [comment]
I agree with a comment above, it is kinda frustrating for an achievement farmer when achievement doesn't work. I also like to farm them in every expansion that I've played so far. Please consider to fix this one for us.. Thank you!

**Comment by Hellbringar** [comment]
I also want to see this one fixed!

**Comment by PanJanuszek** [comment]
Fixed after update

**Comment by Accidentlol** [comment]
Its not fixed still. reopen this.
